cmd/kubeadm/app/apis/bootstraptoken/v1/zz_generated.deepcopy.go

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 package v1
 
 import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
If we added a check that the import alias cannot equal the current pkg name, could we avoid this whole commit?  It's weird to be in pkg "v1" and have an import named "v1".

Reply:
Applied. It looks like conversion-gen was currently importing other packages with the same name as local, so this triggers codegen changes in conversion-gen.  I still think it's a net win, but hadn't realize that until actually trying it out.